dispatcher/frontend-ui/src/views/TaskView.vue

```diff
@@ -157,6 +157,16 @@
         if (this.is_running) {
           return Constants.format_duration_between(first, Constants.now());
         }
+        var dur=0;
+        if(this.task.files&&Object.keys(this.task.files).length>0)
```

Comment:
Note that we bundle sugarjs so you can use it's Array.sum

Reply:
I'll make the required changes